plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/constraintview/util/ConstraintViewDialog.java

```diff
@@ -20,187 +20,44 @@
  */
 package de.ovgu.featureide.fm.ui.views.constraintview.util;
 
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.dialogs.MessageDialogWithToggle;
 import org.eclipse.swt.SWT;
-import org.eclipse.swt.custom.CLabel;
-import org.eclipse.swt.events.SelectionAdapter;
-import org.eclipse.swt.events.SelectionEvent;
-import org.eclipse.swt.layout.RowData;
-import org.eclipse.swt.layout.RowLayout;
-import org.eclipse.swt.widgets.Button;
-import org.eclipse.swt.widgets.Composite;
-import org.eclipse.swt.widgets.Dialog;
 import org.eclipse.swt.widgets.Display;
-import org.eclipse.swt.widgets.Shell;
-import org.eclipse.ui.PartInitException;
-import org.eclipse.ui.PlatformUI;
 
 import de.ovgu.featureide.fm.core.Preferences;
 import de.ovgu.featureide.fm.core.localization.StringTable;
-import de.ovgu.featureide.fm.ui.views.constraintview.ConstraintViewController;
 
 /**
  * A dialog that asks a user to open the constraint view. Its saves the decision in the workspace wide preferences.
  *
- * @author Rosiak Kamil
+ * @author Rosiak Kamil (no functionality of that implementation left)
```

Comment:
Remove the name or the "(no functionality of that implementation left)".

Reply:
Ok.